samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,364 @@
+The latter part of this section presents the structure of this sample application
+
+Isochronous channels
+====================
+
+The Bluetooth LE Isochronous Channels feature is a way of using Bluetooth LE to transfer timebounded data between devices.
+It provides a mechanism that makes sure that multiple sink devices, receiving data from the same source, will render it at the same time.
+Data has a time-limited validity period, at the end of which it is said to expire.
+Expired data which has not yet been transmitted, will be discarded.
+This means that receiver devices only ever receive data which is valid with respect to rules regarding its age and acceptable latency.
+
+The key concepts of isochronous channels are the following:
+
+Topology
+    Isochronous channels can either be used for one-to-many (Broadcast) or one-to-one (Connected).
+    When using connected isochronous streams, data may be time-synchronized between multiple connected devices.
+    Connected isochronous streams can either transfer data from central to peripheral, peripheral to central, or in both directions.
+
+SDU interval and size
+    The SDU interval specificies how often a service data unit (SDU) is transferred.
+    The SDU interval and size as well as the number of streams are limited by the bandwidth of the link layer.
+
+Maximum transport latency and retransmission count
+    Upon setting up an isochronous stream as a central or a broadcaster, the application provides a maximum transport latency and retransmission count.
+    The Bluetooth controller will use the provided parameters to select link layer parameters that satisfy these constraints.
+    This involves selecting an isochronous interval, PDU sizes, and isochronous parameters such as burst number (BN), number of subevents (NSE), and flush timeout (FT) or pretransmission offset (PTO).
+    For more details about how the SoftDevice Controller selects these parameters, see :ref:`Parameter selection <iso_parameter_selection>`.
+
+Presentation delay
+    The receiver(s) of isochronous data may receive their data at different points in time, but need to render/present it synchronously.
+    Presentation delay represents the amount of time that is added to the received timestamp to obtain the presentation time of the data.
+
+* Any of the possible isochronous topologies with unidirectional data transfer.
+  The topology is selected when the application starts.
+* The transmitter of the data sends it over all configured and connected streams.
+  The number of streams is configured with the :kconfig:option:`CONFIG_BT_ISO_MAX_CHAN` Kconfig option.
+* The receiver of the isochronous data receives data from one transmitter only.
+* The maximum transport latency and retransmission count is configured when the application starts.
+* The SDU interval and presentation delay are configured with the :kconfig:option:`CONFIG_SDU_INTERVAL_US` and :kconfig:option:`TIMED_LED_PRESENTATION_DELAY_US` Kconfig options.
```

Comment:
could we add the kconfig references to the sections where you explain the different parameters, above?